src/step_definitions/screenshots.ts

```diff
@@ -0,0 +1,33 @@
+import { Then } from 'cucumber';
+
+Then(/^I take screenshot of the element "([^"]*)" and save as a "([^"]*)"$/, async function(
+  elementName,
+  screenshotName
+) {
+  await browser.imageComparison.saveElement(this.currentPage.getElement(elementName), screenshotName);
+});
+
+Then(/^I take screenshot of the visible part of the page and save as a "([^"]*)"$/, async screenshotName => {
+  await browser.imageComparison.saveScreen(screenshotName);
+});
+
+Then(/^I take full screenshot of the page and save as a "([^"]*)"$/, async screenshotName => {
```

Comment:
Should be `When` we do not validate anything here, we actually performing an action